docs/DeveloperGuide.md

```diff
@@ -395,23 +395,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ------------------------------------------ | ------------------------------------------------- | ------------------------------------------------------------ |
 | `* * *`  | user                                       | create a student's profile                        | keep track of my student's information                       |
 | `* * *`  | user                                       | see the profile of a particular student           | get their information                                        |
-| `* * *`  | user                                       | see a list of all my students' profile            |                                                              |
-| `* * *`  | user                                       | get all the emails of the parent of my students'  | email them reminders for payment                             |
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
+| `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
 | `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
 | `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | user                                       | see a list of all the students profile and sessions|                                                             |
+| `* * *`  | user                                       | get all the emails of the parent of my students'  | email them reminders for payment                             |
+| `* * *`  | user                                       | calculate the monthly fees of a particular student| use the information when collecting monthly fees             |
+| `* * *`  | user                                       | get the monthly fees that I would have received for the past 3 months    | see how well I am doing financially   |
```

Comment:
How about changing "see how well I am doing financially" to "manage my financials better"?

Reply:
Yeap! Sounds good! Have updated in the latest commit.